ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/config/SinkConfig.java

```diff
@@ -361,7 +361,7 @@ private static PubsubMessageToObjectNode getFormat(Env env) {
         return PubsubMessageToObjectNode.Decoded.of();
       case "payload":
         return PubsubMessageToObjectNode.Payload.of(env.getStrings(STRICT_SCHEMA_DOCTYPES, null),
-            env.getString(SCHEMAS_LOCATION), FileInputStream::new).withOpenCensusMetrics();
+            env.getString(SCHEMAS_LOCATION, null), FileInputStream::new).withOpenCensusMetrics();
```

Comment:
ingestion-sink has been updated to support reading table schemas from BigQuery when `schemas_location` is not provided.